src/main/java/equipments/EquipmentType.java

```diff
@@ -0,0 +1,9 @@
+package equipments;
+
+public enum EquipmentType {
```

Comment:
if the different types of equipment are not separated, how can you know which one can go to which factory without looking at the inner implementation?